app/views/conversations/_new.haml

```diff
@@ -1,22 +1,23 @@
 .container-fluid
-  = form_for Conversation.new, html: {class: "form-horizontal form_do_not_clear"}, remote: true do |conversation|
+  = form_for Conversation.new, html: {id: "new-conversation",
+    class: "new-conversation form-horizontal form-do-not-clear"}, remote: true do |conversation|
     .form-group
-      %label#toLabel{for: "contact_ids"}
-        = t(".to")
-      = text_field_tag "contact_autocomplete", nil, class: "form-control"
+      %label#to-label{for: "contacts-search-input"}= t(".to")
+      .clearfix#contacts-tags-list.contacts-tags-list
```

Comment:
Classes should be listed before IDs (.contacts-tags-list should precede #contacts-tags-list)